src/components/logic/StartMenu.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useRef, useContext } from 'react'
+import Quiz from './Quiz'
+import generateChords from '../../chordGenerator'
+import { Session } from '../Context'
+import StartScreen from '../views/StartScreen'
+
+
+export default function Start({ title, round }) {
+  const [session, updateSession] = useContext(Session)
+  const numQs = useRef(5)
+  const [ready, launchQuiz] = useState(false)
+  const [quiz, setQuiz] = useState()
+  const [options, updateOptions] = useState({
+    chordTypes: {triads:true, sevenths:true},
+    roots: {common:true, any:false}
+  })
+
+  // QUESTION: Does this need 'e'? Does it ultimately need to await?
```

Comment:
What is `'e'` ?

Reply:
oops... got rid of it and forgot to delete the comment!